docs/docs/rest-api/public/api/v2/types/pragma.raml

```diff
@@ -39,5 +39,8 @@ annotationTypes:
       Mark a type as internal to indicate, that it is not used in the external API.
       No functional change - only to indicate the use of one type.
       Use only on type level.
-
+  serializeOnly:
+    type: nil
+    description:  |
+      Mark a type as "serialize-only" for which no de-serialization code will be generated.
```

Comment:
it looks like this is only support for object types (vs enum, string, array, map types). please indicate that in the docs